apps/bot-discord/src/config.json

```diff
@@ -334,9 +334,9 @@
       }
     },
     {
-      "regex": "You're offline|Please check your",
+      "regex": "You're offline|Please check your|Couldn't sign in",
       "reply": {
-        "title": "Why does YouTube say, I am offline ❓",
+        "title": "Why does YT ReVanced say I am offline ❓",
```

Comment:
This abbreviationand technically the name should not be used

```suggestion
        "title": "Why YouTube say I am offline after patching ❓",
```

Reply:
"after patching" implies that the issue occurred immediately after patching.

How about `Why does YouTube ReVanced say I am offline?`?

Reply:
That's what it currently is.

Reply:
How about `Why does patched YouTube say I am offline ❓️`